File: Finished algos/algo-anagrams-ii/python/anagram2.py
# Don't forget to run the tests (and create some of your own)
import functools

def anagrams_for(word, list_of_words):
	sortedWord = sorted(list(word.upper()))
	sortedWord = filter((lambda a: a != ' '), sortedWord)
	print(sortedWord)

print(anagrams_for('yes', ['sey', 'no', 'yes', 'seyyes']))

File: Finished algos/algo-anagrams-ii/js/anagram2.js
exports.anagramsFor = function (word, listOfWords) {

    let sortFunction = (elem) => { //Function to sort words
        return elem.toUpperCase().split('').filter(letter => letter !== ' ').sort().join('')
    }

    const mainWord = sortFunction(word)
    // console.log(mainWord)

    let ansArr = []
    let count = 0
    for (let piece of listOfWords) {
        let sortedWord = sortFunction(piece)
        if (mainWord === sortedWord){
            ansArr.push(listOfWords[count])
        }
        count++
    }
    return ansArr
}

//console.log(exports.anagramsFor('Greeek', ['Egr eek', 'gre eke', 'cancer', 'i'])) //=== false)

// Create an array of objects with counts in them from FUNCTION
// 